### ISSUE TITLE
ThreadSelector in Linked Thread Modal does not display threads

### ISSUE THREAD

**Issue by jnaviask** [hicommonwealth/commonwealth#3517]
## Description
Reported by BD. This makes the feature unusable.

![image](https://user-images.githubusercontent.com/1860629/233188338-42014db9-1a64-4e81-8e73-51040c6e9010.png)

## Acceptance Criteria
- The ThreadSelector search should display threads that match the user's search query, and should allow you to link them.

**Comment by masvelio** [comment]
@jnaviask This works fine on local host and on beta (see video).

https://user-images.githubusercontent.com/14819225/233378401-ab2bcf97-ad71-4953-a131-5a0c6780b0f9.mp4

It dose not work though on production, as state in the description. I dont have permission to production logs or DB so can't really figure out what is going on. The response status is `success` (see screenshot) but the array with threads is empty. Payload and endpoint looks good. 

<img width="1662" alt="image" src="https://user-images.githubusercontent.com/14819225/233378884-a0196a2a-c16c-46ba-8d04-d1501e9c8fa1.png">

TLDR: In my opinion it has to be investigated further on production environment
